package.json

```diff
@@ -26,32 +26,32 @@
     }
   },
   "dependencies": {
-    "assertive": "^2.1.1",
-    "debug": "^2.2.0",
+    "assertive": "^3.0.0",
```

Comment:
This driver implicitly exposes some of the assertive API through its `.assert....()` methods - is there anything in the 2 -> 3 breaking change of assertive that would make this a breaking change?